lms/views/api/blackboard/sync.py

```diff
@@ -118,15 +110,20 @@ def to_groups_groupings(self, course, groups):
             type_=Grouping.Type.BLACKBOARD_GROUP,
         )
 
-    def sync_to_h(self, groups):
-        lti_h_svc = self.request.find_service(name="lti_h")
-        group_info = self.request.parsed_params["group_info"]
-        lti_h_svc.sync(groups, group_info)
+    def group_set(self, tool_consumer_instance_guid, resource_link_id):
```

Comment:
Now takes `tool_consumer_instance_guid, resource_link_id`.

Moved to the bottom, closer the the simlar `get_course`.